Misc/NEWS.d/next/Library/2018-10-05-16-01-00.bpo-34547.abbaa.rst

```diff
@@ -0,0 +1,3 @@
+``wsgiref`` now handles abrupt client connection terminations gracefully.
+Previously, long stack traces with Python TypeErrors and AttributeErrors
```

Comment:
This is actually covered in PR #12914, right? I think what we are trying to solve here is to prevent connection-related exceptions like the following to be printed:

```
    [30/Aug/2018 12:10:38] "POST /login/ HTTP/1.1" 200 3964
    Traceback (most recent call last):
      File "d:\python37\Lib\wsgiref\handlers.py", line 138, in run
        self.finish_response()
      File "d:\python37\Lib\wsgiref\handlers.py", line 180, in finish_response
        self.write(data)
      File "d:\python37\Lib\wsgiref\handlers.py", line 274, in write
        self.send_headers()
      File "d:\python37\Lib\wsgiref\handlers.py", line 332, in send_headers
        self.send_preamble()
      File "d:\python37\Lib\wsgiref\handlers.py", line 255, in send_preamble
        ('Date: %s\r\n' % format_date_time(time.time())).encode('iso-8859-1')
      File "d:\python37\Lib\wsgiref\handlers.py", line 453, in _write
        result = self.stdout.write(data)
      File "d:\python37\Lib\socketserver.py", line 796, in write
        self._sock.sendall(b)
    ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
```

(Copied from https://bugs.python.org/msg324375.)

Reply:
Yes. Done.